src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -71,7 +71,20 @@ protected override void ProcessRecord()
 
                         VerbInfo verb = new VerbInfo();
                         verb.Verb = field.Name;
+
+                        FieldInfo aliasField = typeof(VerbAliasPrefixes).GetField(field.Name);
```

Comment:
Suggestion: Refactor this as static method in the associated class. It will provide more flexibility with refactoring, should it be necessary.